lighthouse-core/audits/dobetterweb/no-old-flexbox.js

```diff
@@ -53,28 +53,42 @@ class NoOldFlexboxAudit extends Audit {
 
     // https://www.w3.org/TR/2009/WD-css3-flexbox-20090723/
     // (e.g. box-flex, box-orient, box-flex-group, display: flexbox (2011 version))
-    const propsNames = ['box-flex', 'box-orient', 'box-flex-group', 'display'];
-    const propsNamesWithPrefixes = StyleHelpers.addVendorPrefixes(propsNames);
-    const propsValues = ['box', 'flexbox'];
-    const sheetsUsingDisplayBox = StyleHelpers.filterStylesheetsByUsage(
-        artifacts.Styles, propsNamesWithPrefixes, propsValues);
+    const displayPropResults = StyleHelpers.filterStylesheetsByUsage(artifacts.Styles,
+        'display', StyleHelpers.addVendorPrefixes(['box', 'flexbox']));
+    const otherPropResults = StyleHelpers.filterStylesheetsByUsage(artifacts.Styles,
+        StyleHelpers.addVendorPrefixes(['box-flex', 'box-orient', 'box-flex-group']));
+
+    const sheetsUsingOldFlexbox = displayPropResults.concat(otherPropResults);
+
     const urlList = [];
-    sheetsUsingDisplayBox.forEach(sheet => {
+    sheetsUsingOldFlexbox.forEach(sheet => {
       sheet.parsedContent.forEach(props => {
         const formattedStyleRule = StyleHelpers.getFormattedStyleRule(sheet.content, props);
+
+        let url = sheet.header.sourceURL;
+        if (!url) {
```

Comment:
I also check if it matches the pageUrl if we want to be consistent. Also thoughts on using getDisplayName here?